unstructured_api_tools/pipelines/templates/pipeline_api.txt

```diff
@@ -28,12 +29,35 @@ async def pipeline_1(request: Request, file: UploadFile = File(),
 {% for param in multi_string_param_names %}
 {{param}}: List[str] = Form(default=[]),
 {% endfor %}
+{% if response_type %}
+response_type: str = "{{response_type}}",
+{% endif %}
 ):
 
     text = file.file.read().decode("utf-8")
     response = pipeline_api(text, {% for param in multi_string_param_names %}{{param}}, {% endfor %})
+    
+    media_type = request.headers.get("accept")
+    if media_type != "*/*":
+        response_type = media_type
+    elif "response_type" not in locals():
+        response_type = "*/*"
 
-    return response
+    if (response_type == "application/json" and type(response) != dict) or (
```

Comment:
Can you separate this out into a `_validate_response` function? It'll be too much for the if statement once we add another response type.